### ISSUE TITLE
Feature request - Client LDAP lookup on Open New Ticket

### ISSUE THREAD

**Issue by ppasquin** [osTicket/osTicket#1551]
I'm looking to add the same functionality to the Open a New Ticket page, that exists for the Staff when adding a new user (user-lookup.tmpl.php). I would like the client to be able to type in their username and have osTicket populate their contact information (email, name, phone) from ldap without having to sign in. No need to maintain a user DB when all the information already exists in LDAP.

Thanks,
Paul

**Comment by andrelindner** [comment]
> I'm looking to add the same functionality to the Open a New Ticket page, that exists for the Staff when adding a new user (user-lookup.tmpl.php). I would like the client to be able to type in their username and have osTicket populate their contact information (email, name, phone) from ldap without having to sign in. No need to maintain a user DB when all the information already exists in LDAP.
> 
> Thanks,
> Paul

still an issue? if not, please close this issue.

**Comment by JediKev** [comment]
@andrelindner 

I noticed you spam posted this on many Issue threads and it's kind of rude and annoying tbh. Please, only post on one Issue thread if you're going to post the same thing over and over.

We run our Github project differently than everyone else so there will be things that are different here in this repo than all others. One of these things being the many open issues. We close issues when we are ready to close them as sometimes we keep them open to remind us of certain things or to keep track of debug information, etc. With this being said there are many we _can_ close just haven't gotten around to it. We _might_ setup a day in the future to go over all of them and close the ones that need to be but this will be in the future _if_ we do it at all.

**TLDR;**
Don't spam post...no one likes it. 😁

Cheers.

**Comment by andrelindner** [comment]
> @andrelindner
> 
> I noticed you spam posted this on many Issue threads and it's kind of rude and annoying tbh. Please, only post on one Issue thread if you're going to post the same thing over and over.
> 
> We run our Github project differently than everyone else so there will be things that are different here in this repo than all others. One of these things being the many open issues. We close issues when we are ready to close them as sometimes we keep them open to remind us of certain things or to keep track of debug information, etc. With this being said there are many we _can_ close just haven't gotten around to it. We _might_ setup a day in the future to go over all of them and close the ones that need to be but this will be in the future _if_ we do it at all.
> 
> **TLDR;**
> Don't spam post...no one likes it. 😁
> 
> Cheers.

my post has nothing to do with spam! it's just a reminder for the ticket opener to check the current status after 3-6 years - many questions are already answered - and tbh you are rude if you write something like this without thinking - just for your information, you can close an issue and still look it up for debug informations, do what you want, It looks like a cooperation is not wanted here

good luck - i'm out!

**Comment by JediKev** [comment]
@andrelindner 

I do apologize if I hurt your feelings, that was not intended. I appreciate the feedback and I will work on my wording for next time. The only thing I was trying to say is spam posting (meaning posting the same thing over and over again on multiple threads) is not okay, and is seen to be very annoying to the mods/maintainers (of any repo).

I get what you were trying to say but we have mods for this reason. Like I said, I appreciate the feedback and I will work on my wording for next time.

Cheers.
